tsconfig.json

```diff
@@ -22,5 +22,6 @@
   "atom": {
     "rewriteTsconfig": false
   },
+  "files": ["./node_modules/cordova-plugin-inappbrowser/types/index.d.ts"],
```

Comment:
Maybe we should add `@ionic-native/in-app-browser` that includes the types. So, we don't need to add this to tsconfig

Reply:
Is the plugin `@ionic-native/in-app-browser` an option?